autometa/validation/datasets.py

```diff
@@ -0,0 +1,221 @@
+along with Autometa. If not, see <http://www.gnu.org/licenses/>.
+COPYRIGHT
+
+pulling data from google drive dataset with simulated or synthetic communities
+"""
+
+
+import gdown
+import os
+import sys
+import logging
+import pandas as pd
+
+from autometa.common.utilities import internet_is_connected
+
+logger = logging.getLogger(__name__)
+
+
+def download(
+    community_type: str, community_size: list, file_names: list, dir_path: str
+) -> None:
+
+    """Downloads the files specified in a dictionary.
+
+    Parameters
+    ----------
+    community_type : str
+        specifies the type of dataset that the user would like to download from
+    community_size : list
+        specifies the size of dataset that the user would like to download
+    file_names : list
+        specifies the file(s) that the user would like to download
+    dir_path : str
+        dir_path_size path where the user wants to download the file(s)
+
+    Returns
+    -------
+    None
+        download is completed through gdown
+
+    """
+
+    if community_type == "synthetic" or community_type == "all":
+        sys.exit("Haven't implemented that yet")
+
+    for each_community_size in community_size:
+        df = pd.read_csv("gdown_fileIDs.csv", dtype=str)
+        df = df.query(f'dataset == "{each_community_size}"')
+        dir_path_size = os.path.join(dir_path, each_community_size)
+        # make a new directory
+        if not os.path.exists(dir_path_size):
+            os.mkdir(dir_path_size)
+
+        for file_name in file_names:
+            file_id = df.query(f'file == "{file_name}"')["file_id"].to_list()[0]
+            dir_path_final = os.path.join(dir_path_size, file_name)
+            url = f"https://drive.google.com/uc?id={file_id}"
+
+            gdown.download(url, dir_path_final)
+
+
+def main():
+    import argparse
+    import logging as logger
+
+    logger.basicConfig(
+        format="[%(asctime)s %(levelname)s] %(name)s: %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S %p",
+        level=logger.DEBUG,
+    )
+
+    parser = argparse.ArgumentParser(
+        description="Download a simulated community file from google drive to a specified dir_path_size"
+    )
+    parser.add_argument(
+        "--community_type",
+        help="specify synthetic or simulated communities (currently only simulated is available)",
+        choices=[
+            "synthetic",
+            "simulated",
+            "all",
+        ],
+        required=True,
+    )
+    parser.add_argument(
+        "--community_size",
+        help="specify a community size to download from",
+        choices=[
+            "78Mbp",
+            "156Mbp",
+            "312Mbp",
+            "625Mbp",
+            "1250Mbp",
+            "2500Mbp",
+            "5000Mbp",
+            "10000Mbp",
+            "all",
+        ],
+        required=True,
+        nargs="+",
+    )
+    parser.add_argument(
+        "--file_names",
```

Comment:
- (minor nitpick) replace underscore with hyphen for reduced keystrokes (Accessed the same with argparse)

```suggestion
        "--file-names",
```